lib/master.js

```diff
@@ -257,7 +257,8 @@ Master.prototype._startWorkers = function(remainingWorkers, res) {
                         worker.removeListener('exit', workerExit);
                         self._currentStartingWorker = undefined;
                         self._firstWorkerStarted = true;
-                        res.push(msg.serviceReturns[0]);
+                        res.push(msg.serviceReturns.length > 1 ?
+                            msg.serviceReturns : msg.serviceReturns[0]);
```

Comment:
`serviceReturns` is an array of return values from the different services started in the worker. Currently, you're dropping all but the first one on the floor.